app/src/main/java/com/example/githubusers/model/GithubUsersResponse.java

```diff
@@ -0,0 +1,17 @@
+package com.example.githubusers.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+
+public class GithubUsersResponse {
+
+    @SerializedName("result")
```

Comment:
I believe this(`"result"`) is not the field containing the list of users coming from the `API` kindly ensure the correct value from the API Response